### DISCUSSION TITLE
Feature

### DISCUSSION THREAD

**Discussion by jokepepe** [Ruben2776/PicView#105] [Ideas]
Can we add a keybind to a menu item? Eg I want to keybind "Rename" as "Q". That lets me go through batch photos using W and rename manually with Q. Really useful for large imagesets like weddings or vacation photos.

**Comment by Ruben2776** [comment]
I want to add customizable shortcuts in the future. However it's a difficult thing to implement, with how it's currently set up.

The Image Info window is easily usable for renaming individual images fast, where you can quickly change navigation with the scroll wheel. Though I'm also looking into implementing a bulk rename feature for the batch resize window.

**Comment by Ruben2776** [comment]
Version 2.1 now has customizable shortcuts

**Comment by Yikuten** [comment]
when batch resizing, you have to set the percentage every single time, it would be nice to have it remember the last amount set, or choosing from 100,90,80 etc. like the quality selection. also, percentage does not go like 97.5 or something, it's only 97 or it breaks. these are just some specific ideas that would be nice to have. thank you for your hard work.
